app/scripts/services/Municipalities.svc.js

```diff
@@ -0,0 +1,27 @@
+'use strict';
```

Comment:
This file should have unit tests using $httpBackend to verify that certain URLs are being invoked for each method...but we can circle back on it